deploy.sh

```diff
@@ -3,7 +3,7 @@
 set -ev
 
 # Pull requests and commits to other branches shouldn't try to deploy, just build to verify
-if [[ "$TRAVIS_BRANCH" != master ]]; then
+if [[ "$TRAVIS_BRANCH" != master || "$TRAVIS_PULL_REQUEST" != false ]]; then
```

Comment:
this seems like it previously skipped the deploy on any non-master branch; and now will altos skip the deploy on PRs where the branch is master (like, *from* master to another branch), which in practice never happens. is this intended?

Reply:
No, for PRs, $TRAVIS_BRANCH is the *target* branch; see https://docs.travis-ci.com/user/environment-variables/. (Don't ask me why…)